.github/workflows/arm64.yml

```diff
@@ -96,3 +102,20 @@ jobs:
           docker volume rm -f $(docker volume ls -q) || true
           docker rmi -f $(docker images -aq) || true
           docker system prune -a -f
+  create_jira_issue:
+    name: "Create JIRA issue"
+    runs-on: ubuntu-latest
+    needs: [arm64_health_check]
+    if: always()
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+      - name: "Run the script to create JIRA issue"
```

Comment:
Same here, rename to something like: `Run the JIRA Integration script`

Reply:
Done.